cmake/CMakeLists.txt

```diff
@@ -432,7 +432,11 @@ if(NOT WIN32)
 	LINK_DIRECTORIES(${LINK_DIRECTORIES} /usr/lib /usr/lib64 /usr/local/lib /usr/local/lib64)
 endif()
 
-SET(IFCOPENSHELL_LIBRARIES IfcParse IfcGeom)
+if(BUILD_SHARED_LIBS)
```

Comment:
Ok, but `IFCOPENSHELL_LIBRARIES` is also used in the `INSTALL()` statement. Meaning that IfcParse is not installed with `BUILD_SHARED_LIBS`.